### PR TITLE
prepare for community site

### PR DESCRIPTION
**Pull request by amilan17**
Removed or replaces github links; some editorial updates

### CODE REVIEW

**Comment by amilan17** [comment]
I also recommend changing this sentence from: 

"Alongside the topic hierarchy, a standard notification message format has been defined. Using such a format will allow the entire WIS2 ecosystem: Data publisher, Data User, Global Services to ensure robustness, effectiveness of WIS2 as a unified weather, climate, water data exchange platform."

to: 
"The standard notification message format ensures that the WIS2 ecosystem (data publisher, data user, and global services) is a robust, effective, and unified data exchange platform for weather, climate, and water. "

File: WIS2_Message_Format_README.adoc [hunk 1]
```diff
@@ -273,13 +272,13 @@ The _links_ array includes one or more URLs to give access to data.
 .Detail of the *links* array
 [cols="1,1,1,1,2"]
 |===
-| Key | Value(s) | Example | Status | Description
+| Key | Value(s) | Example | Status | Deßscription
```
Comment (tomkralidis) [inline WIS2_Message_Format_README.adoc:275]: Description

**Review by tomkralidis — CHANGES_REQUESTED** [review]

**Comment by tomkralidis** [comment]
In addition to minor change request, +1 to your suggested update @amilan17

**Comment by amilan17** [comment]
@tomkralidis I think this is ready to merge to master.

**Comment by tomkralidis** [comment]
@amilan17 I made a commit to `editorial` branch as follows:

- update the nullable geometry in the schema (https://github.com/wmo-im/wis2-notification-message/commit/c342aa5cb1e2b9bfa630091aa6a5f66f4be0be5b)
- add a geometry example with elevation (https://github.com/wmo-im/wis2-notification-message/commit/e7ad1b9c7343d9b5e194a845edad6aaf537763cd)

Let me know if these make sense then we can merge.  Thanks!

**Comment by amilan17** [comment]
@tomkralidis - looks good to me.

**Review by tomkralidis — APPROVED** [review]

**Pull request merged**